### PR TITLE
Streamline settings

### LINKED ISSUE #15: Remove `no-default-files`?
I'm not sure it makes sense for us to expose this CLI flag as an action setting -- the user can simply choose not to use any artifacts/assets if they don't care about the default file outputs.

Thoughts @tetsuo-cpp?

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
Closes #15

### CODE REVIEW

**Comment by tetsuo-cpp** [comment]
Here I'm removing a few redundant settings. I also added an `identity-token` setting since users might not want to use the GitHub Actions credential.

File: README.md [hunk 1]
```diff
@@ -319,6 +281,25 @@ This setting only applies if `verify` is set to `true`.
     verify-cert-email: john.smith@example.com
 ```
 
+### `verify-oidc-issuer`
```
Comment (tetsuo-cpp) [inline README.md:284]: We only support getting the GitHub Actions OIDC token OR supplying our own token. Querying the OIDC issuer and getting a token in real time is NOT supported so let's make that clear in the documentation.

File: action.py [hunk 2]
```diff
@@ -72,6 +72,10 @@ def _fatal_help(msg):
 if _DEBUG:
     sigstore_python_env["SIGSTORE_LOGLEVEL"] = "DEBUG"
 
+identity_token = os.getenv("GHA_SIGSTORE_PYTHON_IDENTITY_TOKEN")
+if identity_token != "":
+    sigstore_sign_args.extend(["--identity_token", identity_token])
```
Comment (woodruffw) [inline action.py:77]: I think this should be:

```suggestion
    sigstore_sign_args.extend(["--identity-token", identity_token])
```

Comment (tetsuo-cpp) [reply action.py:77]: Good catch, I was about to test that in #23 haha.

**Comment by tetsuo-cpp** [comment]
Sorry, the testing PR was actually pointing at this one. I did that because the tests relied on the new `identity-token` setting.

**Comment by woodruffw** [comment]
> Sorry, the testing PR was actually pointing at this one. I did that because the tests relied on the new `identity-token` setting.

No problem, it was my mistake. Do you want to roll that other PR back?

**Review by woodruffw — APPROVED** [review]
LGTM!

**Comment by tetsuo-cpp** [comment]
> > Sorry, the testing PR was actually pointing at this one. I did that because the tests relied on the new `identity-token` setting.
> 
> No problem, it was my mistake. Do you want to roll that other PR back?

I'm not too worried about it. I'm happy to just merge this if you agree.

**Comment by woodruffw** [comment]
> I'm not too worried about it. I'm happy to just merge this if you agree.

Yep, go for it.

**Pull request merged**